package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "node-ember-cli-deploy-redis",
-  "version": "0.0.1",
-  "description": "A node middleware to serve EmberJS apps deployed by ember-cli-deploy",
+  "version": "1.0.0",
```

Comment:
Good idea. I wanted to bump this because this is a breaking change from 0.0.1, but there aren't enough users of this yet.